src/Permissions/Backend/App.vala

```diff
@@ -108,11 +108,14 @@ public class Permissions.Backend.App : GLib.Object {
                 }
             }
 
-            var s = new Backend.PermissionSettings (key, standard) {
-                enabled = enabled
-            };
+            // Only add settings that are actually overridden
```

Comment:
I'm -1 on this. I need to allow certain applications home folder access. One of them is Telegram, otherwise files drag-and-drop doesn't work. And there are bunch of such applications on Flathub.